code/game/turfs/open/space/space.dm

```diff
@@ -22,6 +22,8 @@
 	bullet_bounce_sound = null
 	vis_flags = VIS_INHERIT_ID //when this be added to vis_contents of something it be associated with something on clicking, important for visualisation of turf in openspace and interraction with openspace that show you turf.
 
+	force_no_gravity = TRUE
```

Comment:
This might make breaches more lethal than previously. Are you sure about this?

Reply:
what?

Reply:
this is a replacement for has_gravity() behavior where it typechecks the loc to see if its a space turf so it can return no gravity (unless the object has forced gravity), now it just checks for this var so any turf can do this.